jib-maven-plugin/src/main/java/com/google/cloud/tools/jib/maven/BuildDockerMojo.java

```diff
@@ -70,7 +70,7 @@ public void execute() throws MojoExecutionException {
       ImageReference targetImage =
           ConfigurationPropertyValidator.getGeneratedTargetDockerTag(
               getTargetImage(),
-              mavenJibLogger,
+              mavenProjectProperties.getEventEmitter(),
```

Comment:
Who's setting an `EventEmitter` and where? Is logging actually working? I believe there should exist code that adds a real event handler for `LogEvent` that actually stream logs to `AbstractMojo.getLog()`. I don't see such code.

Reply:
Oh, maybe the log handler is already added somewhere before this PR?

Reply:
Yeah, that happens in `LogEventHandler` I think

Reply:
Yeah, I see `makeEventEmitter()` in `GradleProjectProperties` and `MavenProjectProperties` adds the log event handler. We're good.